Jetcaster/wear/src/main/java/com/example/jetcaster/ui/library/LibraryScreen.kt

```diff
@@ -62,166 +75,205 @@ fun LibraryScreen(
 ) {
     val uiState by libraryScreenViewModel.uiState.collectAsState()
 
-    val columnState = rememberResponsiveColumnState(
-        contentPadding = ScalingLazyColumnDefaults.padding(
-            first = ScalingLazyColumnDefaults.ItemType.Text,
-            last = ScalingLazyColumnDefaults.ItemType.Chip,
-        ),
+    val contentPadding = rememberResponsiveColumnPadding(
+        first = ColumnItemType.ListHeader,
+        last = ColumnItemType.Button,
     )
 
-    when (val s = uiState) {
-        is LibraryScreenUiState.Loading ->
-            LoadingScreen(
-                modifier = modifier,
-            )
-        is LibraryScreenUiState.NoSubscribedPodcast ->
-            NoSubscribedPodcastScreen(
-                columnState = columnState,
-                modifier = modifier,
-                topPodcasts = s.topPodcasts,
-                onTogglePodcastFollowed = libraryScreenViewModel::onTogglePodcastFollowed,
-            )
+    val columnState = rememberTransformingLazyColumnState()
+    ScreenScaffold(
+        scrollState = columnState,
+        contentPadding = contentPadding,
+        modifier = modifier,
+    ) { contentPadding ->
+        when (val s = uiState) {
```

Comment:
consider naming s something more descriptive

Reply:
why is this when statement using different syntax than the other ones?

Reply:
this is not part of these set of changes for Material 3, this was already here. Can we keep improvements for a follow up PR to close up on the Material 3 migration?